src/tito/builder/main.py

```diff
@@ -246,9 +250,8 @@ def rpm(self):
             err = sys.exc_info()[1]
             msg = str(err)
             if re.search('Failed build dependencies', err.output):
-                cmd = "dnf builddep %s" if package_manager() == "dnf" else "yum-builddep %s"
-                msg = "Please run '%s' as root." % \
-                    cmd % find_spec_file(self.relative_project_dir)
+                cmd = self.package_manager.builddep(find_spec_file(self.relative_project_dir))
+                msg = "Please run '%s' as root." % cmd
```

Comment:
Am I following correctly, if it's a missing build requires, we'll ask you to run a command, if it's a missing requires when we try to --test --install, we'll try to install it for you?